pipelines/chess/__init__.py

```diff
@@ -86,10 +93,8 @@ def players_games(players: List[str], start_month: str = None, end_month: str =
     # get archives in parallel by decorating the http request with defer
     @dlt.defer
     def _get_archive(url: str) -> List[TDataItem]:
-        print(f"Getting archive from {url}")
```

Comment:
ohhh please leave this one. it is very useful when showing/training someone

Reply:
done (was removed in separated PR)